client.go

```diff
@@ -64,3 +64,12 @@ Close client connection
 func (c *Client) Close() {
 	closeChannel(&c.Channel, &c.methods)
 }
+
+func (c *Client) ConnectNamespace(namespace string) error {
```

Comment:
Can we add a comment explaining what this is doing?

Is it setting the default namespace for the underlying connection? Or does it just make this connection eligible to call `Emit` with this namespace in the future?

Reply:
Default is no namespace. It allows multiple namespaces to be connectioned in the same session. And you do need to connect to one specific namespace before sending request to that particular namespace.
Comments added.